maven-core/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -122,8 +122,8 @@ under the License.
           <field-name>_cipher</field-name>
         </requirement>
         <requirement>
-          <role>org.sonatype.plexus.components.sec.dispatcher.PasswordDecryptor</role>
```

Comment:
This is a typo in [Plexus Sec Dispatcher](https://github.com/sonatype/plexus-sec-dispatcher/blob/master/src/main/java/org/sonatype/plexus/components/sec/dispatcher/PasswordDecryptor.java#L25) we can't fix here.